### PR TITLE
demo pull request

### PR DESCRIPTION
**Pull request by rsomcio**
### TODO
- [x] demo mark down
- [x] demo @mentions
- [x] demo cross-links
- [x] demo emoji

### CODE REVIEW

**Comment by rsomcio** [comment]
@githubteacher testing mentions?

**Comment by rsomcio** [comment]
:100:

**Comment by rsomcio** [comment]
:facepunch: :+1: :hankey:

File: rayso-hello-world.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print "hello world"
```
Comment (JosephineGitHub) [inline rayso-hello-world.py:3]: awesome change! looks good.

Comment (rsomcio) [inline rayso-hello-world.py:3]: thanks

**Comment by rsomcio** [comment]
#### something

`#1 hello`

**Pull request merged**